src/uikit.cs

```diff
@@ -331,7 +331,17 @@ interface NSAttributedStringAttachmentConveniences {
 	[DisableDefaultCtor]
 	[Abstract] // abstract class that should not be used directly
 	[BaseType (typeof (NSObject))]
+#if IOS && IOS17_5_OR_GREATER
+	interface UIFeedbackGenerator : UIInteraction {
+#else
```

Comment:
`IOS17_5_OR_GREATER` isn't ever defined, so this if block is never compiled.

Reply:
Ah then I’ll just leave the iOS part, I kinda remember you introduced that feature, is that a thing? Or I got it wrong @rolfbjarne?

Reply:
I don't remember anything in this area, there's no way to do this on a per-OS version.

Since UIFeedbackGenerator is only available on iOS anyways, I think it would make sense to remove the `#if IOS` condition as well.

Reply:
@rolfbjarne alright, all fixed.